modules/nw-ipfailover-configuration.adoc

```diff
@@ -67,7 +67,7 @@ spec:
         node-role.kubernetes.io/worker: ""
       containers:
       - name: openshift-ipfailover
-        image: quay.io/openshift/ose-keepalived-ipfailover:latest
+        image: quay.io/openshift/ose-keepalived-ipfailover
```

Comment:
In bug description it says the correct  one is `origin-keepalived-ipfailover` not `ose-keepalived-ipfailover`.

Reply:
@rfredette could you help confirm if this right image for end user?

Reply:
@lihongan and @rfredette  My bad! I didn't update it to origin-keepalived-ipfailover. Apologies from my side!

Reply:
@lihongan and @rfredette. I have fixed it. Please review and let me know if you're okay with it.